hugegraph-api/src/main/java/com/baidu/hugegraph/auth/WsAndHttpBasicAuthHandler.java

```diff
@@ -114,19 +113,16 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
                     sendError(ctx, msg);
                     return;
                 }
-                byte[] userPass = null;
+                byte[] userPass;
                 try {
                     final String encoded = header.substring(basic.length());
                     userPass = this.decoder.decode(encoded);
-                } catch (IndexOutOfBoundsException iae) {
-                    sendError(ctx, msg);
-                    return;
-                } catch (IllegalArgumentException iae) {
+                } catch (IndexOutOfBoundsException | IllegalArgumentException iae) {
                     sendError(ctx, msg);
                     return;
                 }
                 String authorization = new String(userPass,
-                                                  Charset.forName("UTF-8"));
+                        StandardCharsets.UTF_8);
```

Comment:
we use the `hugegraph-style` in the wiki (not java original style)

align the param with the highly priority